src/main/scala/org/globalforestwatch/summarystats/afi/AFiAnalysis.scala

```diff
@@ -48,6 +48,8 @@ object AFiAnalysis extends SummaryAnalysis {
     val summaryDF = AFiAnalysis.aggregateResults(
         AFiDF
         .getFeatureDataFrame(summaryRDD, spark)
+        .filter($"location_id" =!= -2)
```

Comment:
I think actually we'd want to filter out -2 before we run the analysis, since the first time a user runs this, -2 will be the same as -1, a huge dissolved polygon. You could run this same filter but for the input feature DF.